### ISSUE TITLE
Component menu missing items after update from 3.6.5 to 3.7

### ISSUE THREAD

**Issue by euoceo** [joomla/joomla-cms#15728]
I saw one thread/bug regarding this, but the solution did not apply to me. After updating to 3.7 from 3.6.5 a number of items under the component menu are missing(redirect is what I need). I also had to re-discover several Joomla built-in items as previously noted, which fixed a couple issues, but not the Component menu.![screen shot 2017-05-02 at 00 01 16](https://issues.joomla.org/uploads/1/aca7e978aaa7a3b8726e9195a8da1d9e.png)

See attached picture.

System Information
Setting	Value
 
Database Version	5.5.5-10.0.30-MariaDB
Database Collation	latin1_swedish_ci
Database Connection Collation	utf8mb4_general_ci
PHP Version	7.0.18
Web Server	Apache
WebServer to PHP Interface	cgi-fcgi
Joomla! Version	Joomla! 3.7.0 Stable [ Amani ] 25-April-2017 15:36 GMT
Joomla! Platform Version	Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.81 Safari/537.36

**Comment by franz-wohlkoenig** [comment]
closed as Duplicate Report #15719<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15728">issues.joomla.org/tracker/joomla-cms/15728</a>.</sub>

**Comment by Tuscany58** [comment]
Today I had the same issue on some of my website. They all have been updated to Joomla 3.8 as soon as the update was available. Now some days later - over night - component items in admin menu disappeared (beside some Joomla core components) 
Google led me to this discussion, and I had a look to the table #_menu where I firstly saw some the components listed  as menutype "main" and some as "menu".
As I opened the database a second time all the "menu" entries were changed to "main" (magic?)
"main" seemed to be the right menutype but components were still hidden in the admin menu.
comparing the entries field by field I found that the field "access" has been set to "5" (access level for guests)
With this code I changed all entries in access field for menutype main to 0 "et voila" all the components showed up in the admin menu
`UPDATE '#-menu' SET 'access' = 0 WHERE 'access' =5 AND 'menutype='main'`

It works for me now, but will this be a stable solution?
